taui/src/components/route-card.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 import Icon from '@conveyal/woonerf/components/icon'
-import message from '@conveyal/woonerf/message'
 import React from 'react'
+import message from '@conveyal/woonerf/message'
```

Comment:
Apparently the linter thinks this is fine, but did you intend to separate the `@conveyal` imports here?